README.md

```diff
@@ -18,10 +18,11 @@ Launch with:
 
 If you have a [Docker daemon running](https://docs.docker.com/installation/), e.g. reachable on `localhost`, start a container with:
 
-    $ docker run --rm -itP -v "$(pwd):/notebooks" jupyter/notebook
+    $ docker run --rm -it -p 8888:8888 -v "$(pwd):/notebooks" jupyter/notebook
 
 In your browser open the URL `http://localhost:8888/`.
-All notebooks from your session will be saved in the current directory.
+All notebooks from your session will be saved in the current directory. In reality, they are save inside the container at `/notebook`.
```

Comment:
your example may be well for demonstration purposes. but it's hardly useful for daily usage.

my point is: this is a documentation of the Docker image. that image includes a volume declaration. for non-demonstration-usages it is essential to mention this volume explicitly.